src/main/java/org/spideruci/tacoco/MavenBuildProbe.java

```diff
@@ -103,14 +105,14 @@ public BuilderType getBuilderType() {
 	public String getClasspath(){
 		try{
 			if(classpath != null) return classpath;
-			if(!new File(targetDir+"/tacoco.cp").exists()) {
+			if(!new File(targetDir + File.separator + "tacoco.cp").exists()) {
                 MavenCli mavenCli = new MavenCli();
-                mavenCli.doMain(new String[] {"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
+                mavenCli.doMain(new String[]{"dependency:build-classpath", "-Dmdep.outputFile=tacoco.cp"}, targetDir,
                         System.out, System.out);
 			}
 			classpath = new String(Files.readAllBytes(Paths.get(targetDir,"tacoco.cp")))
-					+":"+ targetDir + "/target/test-classes"
-					+":"+ targetDir + "/target/classes";
+					+ File.pathSeparator + StringUtils.join(new String[]{targetDir, "target", "test-classes"}, File.separator)
+					+ File.pathSeparator + StringUtils.join(new String[]{targetDir, "target", "classes"}, File.separator);
```

Comment:
Use a StringBuffer, Guava or write a custom utility method that does String Join